Production_scripts/Create_social_care_lookups/lookup_care_home_name.R

```diff
@@ -0,0 +1,108 @@
+#####################################################
+# Care Home Name Lookup
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - SLF Care Home Name Lookup
+# Description - Cleans up Care Home Names
+#####################################################
+
+
+# Load packages
+library(dplyr)
+library(dbplyr)
+library(createslf)
+library(phsmethods)
+
+
+year <- check_year_format("1920")
+
+
+# Read in data---------------------------------------
+
+
+ch_lookup <- readxl::read_xlsx(get_slf_ch_name_lookup_path())
+
+
+# Data Cleaning---------------------------------------
+
+ch_clean <- ch_lookup %>%
+  # correct postcode formatting
+  mutate(AccomPostCodeNo = postcode(AccomPostCodeNo)) %>%
```

Comment:
```suggestion
  mutate(AccomPostCodeNo = format_postcode(AccomPostCodeNo)) %>%
```

This function has been renamed in the latest version of `phsmethods`, can you make sure you've updated it then re-check this bit please?